src/components/ui/PrevStepButton.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Button } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+
+type PreviousStepButtonProps = {
+  onSubmit: (action: string) => void
+}
+
+const PrevStepButton = (props: PreviousStepButtonProps) => {
+  const { onSubmit } = props
```

Comment:
maybe makes sense to change the naming, bc it's not submit button - onHadleStepBack or smth similar

Reply:
Agree. Changed into stepBackHandler